### ISSUE TITLE
Accessibility - Font Size Changes

### ISSUE THREAD

**Issue by kgar** [kgar/foundry-vtt-tidy-5e-sheets#4]
Tidy 5e Sheets should be able to scale based on the root font size. By default, this font size is set to 16px by Foundry, but if the user increases it to 32px, for example, Tidy 5e Sheets should scale up comfortably. Its content should be readable, and its layout should not break in unexpected ways.

## References

https://discord.com/channels/732325252788387980/1116078321067892796/1160617084955394088

From discord user fallayn:
> btw my players with worse eyesight just noted that some parts of the sheet break pretty badly if you increase the font sizes in Foundry's settings (like the currency section)

**Comment by kgar** [comment]
This is already supported in Foundry with font scaling, plus the ability to zoom the browser itself.
